core-lib/Benchmarks/AsyncHarness.ns

```diff
@@ -120,9 +120,21 @@ class AsyncHarness usingPlatform: platform = Value (
     )
 
     printRun: runTime = (
-      | stats = system traceStatistics. |
-      (name + ': trace size:    ' + (stats at: 1) + 'byte') println.
-      (name + ': external data: ' + (stats at: 2) + 'byte') println.
+      | stats = system traceStatistics.
+        memstats = system memoryStatistics.
+      |
+
```

Comment:
Can we avoid printing anything for which the value is 0?
I am not sure all our tools work with this, but it would minimize the output.

Reply:
I implemented it for the trace stats, MemoryStats have to be enabled with a flag so that shouldn't be necessary